packages/synapse-react-client/src/lib/containers/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -224,33 +137,6 @@ export default function RequestDataAccessStep2(
     dataAccessRequest?.concreteType ===
     'org.sagebionetworks.repo.model.dataaccess.Renewal'
 
-  const batchFilesRequest: BatchFileRequest = useMemo(() => {
```

Comment:
This only fetched the file handle for the DUC template. I extracted a generic `DownloadDocumentTemplate` component, which will handle this fetching internally.